packages/flutter_test/lib/src/window.dart

```diff
@@ -154,8 +154,13 @@ class TestWindow implements ui.SingletonFlutterWindow {
   }
 
   @override
-  // ignore: unnecessary_non_null_assertion
-  ui.Locale get locale => _localeTestValue ?? platformDispatcher.locale!;
+  ui.Locale? get locale {
+    if (_localeTestValue !=null) {
+      return _localeTestValue!;
+    }
+    return platformDispatcher.locale;
+  }
+
```

Comment:
I think Making `ui.Locale get` -> `ui.Locale? get` should enable the oneline version of this to work:

```dart
ui.Locale? get locale => _localeTestValue ?? platformDispatcher.locale;
```

Reply:
(I don't understand why:

```dart
ui.Locale get locale => _localeTestValue ?? platformDispatcher.locale;
```

is throwing the [errors above](https://github.com/flutter/flutter/pull/70242#issuecomment-725035319), however!)

Reply:
Yes it also didn't produce the error when I run the integration tests:

```
flutter drive --target=test_driver/platform_messages_integration.dart -d web-server --browser-name=chrome --local-engine=host_debug_unopt
```

But it broke the smoke tests: https://cirrus-ci.com/task/4968866939928576?command=main#L0